SingularityUI/app/templates/dashboardTable/dashboardStarred.hbs

```diff
@@ -2,12 +2,12 @@
     <table class="table table-striped">
```

Comment:
should this table also have the `sortable-theme-bootstrap" data-sortable` attributes?